### PR TITLE
minor changes/edits made to sections 1-5

### PR DESCRIPTION
**Pull request by relaxedtomato**

### CODE REVIEW

File: book/core-concepts/styles.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
-React Native allows you to style components using CSS properties as inline styles. For layout only the flexbox module and absolute positioning is available. The style properties are split into five categories:
+React Native allows you to style components using limited CSS properties as inline styles. For layout only the flexbox module and absolute positioning is available. The [style properties](https://facebook.github.io/react-native/releases/next/docs/style.html#supported-properties) are split into five categories:
 
-1. [View Properties](https://facebook.github.io/react-native/docs/view.html#style)
-2. [Image Properties](https://facebook.github.io/react-native/docs/image.html#style)
-3. [Text Properties](https://facebook.github.io/react-native/docs/text.html#style)
-4. [Flex Properties](https://facebook.github.io/react-native/docs/flexbox.html#content)
-5. [Transform Properties](https://facebook.github.io/react-native/docs/transforms.html#content)
+1. View Properties
```
Comment (winkerVSbecks) [inline book/core-concepts/styles.md:5]: Why remove the links?

File: book/list-view/index.md [hunk 2]
```diff
@@ -115,7 +115,7 @@ const MediaObject = ({ index, action, text, imageUrl }) => {
 
 ```
 
-Again, don't forget to do housekeeping like defining your styles and properly exporting your component as a module here. We've also used a bgColor function above that you can define thusly:
+Again, don't forget to do housekeeping like defining your styles and properly exporting your component as a module here. We've also used a bgColor function above that you can define:
```
Comment (winkerVSbecks) [inline book/list-view/index.md]: typo? `thusly` should be in there imo

File: book/navigation/index.md [hunk 3]
```diff
@@ -10,7 +10,10 @@ See the [navigator comparison documentation for more information.](https://faceb
 In this workshop we will be working with a third option: `NavigationExperimental`.
 `NavigationExperimental` differs from the `Navigator` component in that it attempts
 to be more like Redux using a single-direction flow of data and reducers to manage
-it's state.
+it's state.
```
Comment (winkerVSbecks) [inline book/navigation/index.md]: typo `its`

File: book/navigation/navigation-experimental-details.md [hunk 4]
```diff
@@ -4,10 +4,10 @@ The `NavigationRootContainer` is the component that handles your application's n
 Normally you would pass a reducer to this component and it will handle returning a new navigation
 state when it's onNavigate method has been called.
 
-There is also the NavigationContainer.create method, which acts similar to react-redux's `connect` method.
+There is also the `NavigationContainer.create` method, which acts similar to react-redux's `connect` method.
 It does not pull in any state however, but does pass the component it wraps an `onNavigate` function.
```
Comment (winkerVSbecks) [inline book/navigation/navigation-experimental-details.md]: minor edit

```
It does not pull in any state however, it passes an `onNavigate` function to the component it wraps.
```

**Comment by winkerVSbecks** [comment]
couple of minor suggestions otherwise :shipit:

File: book/core-concepts/styles.md (hunk 1 above)
Comment (relaxedtomato) [inline book/core-concepts/styles.md:5]: Just included a parent link instead.

Comment (winkerVSbecks) [inline book/core-concepts/styles.md:5]: 👍

**Pull request merged**